util/no_destructor.h

```diff
@@ -33,12 +28,11 @@ class NoDestructor {
   NoDestructor& operator=(const NoDestructor&) = delete;
 
   InstanceType* get() {
-    return reinterpret_cast<InstanceType*>(&instance_storage_);
+    return reinterpret_cast<InstanceType*>(+instance_storage_);
   }
 
  private:
-  typename std::aligned_storage<sizeof(InstanceType),
-                                alignof(InstanceType)>::type instance_storage_;
+  alignas(InstanceType) unsigned char instance_storage_[sizeof(InstanceType)];
```

Comment:
I think we used the more complicated form because `alignas` wasn't working as intended in some C++ compiler we supported. 

It seems like this is no longer a problem. Chromium uses `alignas` now -- https://source.chromium.org/chromium/chromium/src/+/main:base/no_destructor.h;l=94;drc=ded7717f5252a0c9d9de56bc22483b195d1801e0